packages/react-components/react-provider/src/components/FluentProvider/useFluentProvider.ts

```diff
@@ -89,6 +93,15 @@ export const useFluentProvider_unstable = (
       dir,
       ref: useMergedRefs(ref, useFocusVisible<HTMLDivElement>({ targetDocument })),
     }),
+
+    serverStyleProps: {
+      cssRule: rule,
+      rendererAttributes: {
```

Comment:
```suggestion
      rule,
      attributes: {
```

nit: It's not `rendererAttributes` for sure 🐱